mne/viz/epochs.py

```diff
@@ -88,6 +88,11 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         and is added to the image. It is typically useful to display reaction
         times. Note that it is defined with respect to the order
         of epochs such that overlay_times[0] corresponds to epochs[0].
+    gfp : bool
+        If True, plot the Global Field Power (one figure per channel type).
+        The GFP is calculated on a single-trial basis for the image, but the
+        time series plot below shows the GFP of the average across epochs,
```

Comment:
just curious, why on the average of epochs instead of the average of individual GFPs?